manager/integration/tests/test_scheduling.py

```diff
@@ -432,11 +432,10 @@ def test_replica_auto_balance_node_least_effort(client, volume_name):  # NOQA
     And set `replica-auto-balance` to `least_effort`.
     And disable scheduling for node-2.
         disable scheduling for node-3.
-    And create a volume.
+    And create a volume with 6 replicas.
```

Comment:
Why we need 6 replicas instead of the default 3?

Reply:
I want to see a clear test result, if using default 3 then is unclear to see if the result is from least effort of best effort.